tests/robustness/validate/validate_test.go

```diff
@@ -37,6 +37,8 @@ func TestValidate(t *testing.T) {
 			path := filepath.Join(testdataPath, file.Name())
 			reports, err := report.LoadClientReports(path)
 			assert.NoError(t, err)
+			// TODO(MadhavJivrajani): update test here to include WithPrevKV()
+			// Will probably need to repopulate testdata.
```

Comment:
@serathius apart from https://github.com/etcd-io/etcd/pull/17352#discussion_r1485965942, I could use your input on this ^

I'm still having some difficulty running a robustness test which is on a cluster of size 3, even with the `LowTraffic` profile, this is what I get:
```
    logger.go:130: 2024-02-12T15:42:14.448+0530	INFO	Recorded operations	{"operationCount": 190}
    logger.go:130: 2024-02-12T15:42:14.448+0530	INFO	Average traffic	{"qps": 53.50818790796354}
    traffic.go:105: Requiring minimal 100.000000 qps for test results to be reliable, got 53.508188 qps
```
It is worth noting that this is kinda flaky because sometimes it does end up working. What I am wondering is why that is the case, I don't see flakiness in the CI for this type of test and my system seems to have the right specs to run this as well
```
❯ system_profiler SPHardwareDataType
Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Processor Name: 8-Core Intel Core i9
      Processor Speed: 2.4 GHz
      Number of Processors: 1
      Total Number of Cores: 8
      L2 Cache (per Core): 256 KB
      L3 Cache: 16 MB
      Hyper-Threading Technology: Enabled
      Memory: 32 GB
```

Reply:
Not sure, we have the qps validation to make sure that tests serve their purpose and have enough QPS to trigger historical issues. 100 is pretty low qps to ensure it runs on CI. Don't think this is CPU limitation as I get 300 qps on my laptop 13 inch in power saver mode.

Don't know how to help you in this case, you can lower the qps min target locally, and we can validate the change fully on CI.

Reply:
I regenerated the test data by changing the min qps to 50